tests/text/test_sacrebleu.py

```diff
@@ -0,0 +1,73 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+
+from tests.text.helpers import TextTester
+from torchmetrics.functional.text.sacrebleu import sacrebleu_score
+from torchmetrics.text.sacrebleu import SacreBLEUScore
+from torchmetrics.utilities.imports import _SACREBLEU_AVAILABLE
+
+if _SACREBLEU_AVAILABLE:
+    from sacrebleu.metrics import BLEU
+
+# example taken from https://github.com/mjpost/sacrebleu
+REFERENCES = [
+    # First set of references
+    ["The dog bit the man.", "It was not unexpected.", "The man bit him first."],
+    # Second set of references
+    ["The dog had bit the man.", "No one was surprised.", "The man had bitten the dog."],
+]
+
+HYPOTHESES = ["The dog bit the man.", "It wasn't surprising.", "The man had just bitten him."]
+
+TOKENIZERS = ["none", "13a", "zh", "intl", "char"]
```

Comment:
```suggestion
REFERENCES = (
    # First set of references
    ("The dog bit the man.", "It was not unexpected.", "The man bit him first."),
    # Second set of references
    ("The dog had bit the man.", "No one was surprised.", "The man had bitten the dog."),
)

HYPOTHESES = ("The dog bit the man.", "It wasn't surprising.", "The man had just bitten him.")

TOKENIZERS = ("none", "13a", "zh", "intl", "char")
```
just to avoid random chnages